include/alpaka/atomic/AtomicOaccBuiltIn.hpp

```diff
@@ -0,0 +1,388 @@
+/* Copyright 2020 Jeffrey Kelling
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#ifdef ALPAKA_ACC_ANY_BT_OACC_ENABLED
+
+#if _OPENACC < 201306
+    #error If ALPAKA_ACC_ANY_BT_OACC_ENABLED is set, the compiler has to support OpenACC xx or higher!
+#endif
+
+#include <alpaka/atomic/Traits.hpp>
+#include <alpaka/atomic/Op.hpp>
+
+namespace alpaka
+{
+    //#############################################################################
+    //! The OpenACC accelerator's atomic ops.
+    //
+    //  Atomics can be used in the blocks and threads hierarchy levels.
+    //  Atomics are not guaranteed to be safe between devices or grids.
+    class AtomicOaccBuiltIn
+    {
+    public:
+        //-----------------------------------------------------------------------------
```

Comment:
Why do we have these lines here?

Reply:
Copy from OpenMP.

Reply:
That is not a reason for keeping them. But do as you want.

Reply:
The OpenMP atomics file has the same lines in the same place, this minimizes the diff to OpenMP.